src/constants.ts

```diff
@@ -72,7 +72,8 @@ export const sectionsDescription = `- by default, typing a query will fetch data
 - the order of the objects in the \`sections\` array determines the order of the results
 - each autocomplete section object must have a \`indexSectionName\`
 - each recommendation section object must have a \`podId\`
-- each custom section object must have a \`displayName\`
+- recommendation section displayName if passed will override the displayName set by Merchandisers
```

Comment:
wdyt of 

`passing displayName in the sectionConfiguration will override the displayName set by Merchandisers on the Customer Dashboard`

Reply:
I approve